src/include/kernel/tokens.h

```diff
@@ -22,6 +22,7 @@
 #define SOF_TPLG_KCTL_ENUM_ID	257
 #define SOF_TPLG_KCTL_BYTES_ID	258
 #define SOF_TPLG_KCTL_SWITCH_ID	259
+#define SOF_TPLG_KCTL_BYTES_READONLY_ID 260
```

Comment:
It's probably best to add 
```c
#define SOF_TPLG_KCTL_BYTES_VOLATILE_RO 260
#define SOF_TPLG_KCTL_BYTES_VOLATILE_RW 261
```
This way it's clear as volatile RO or RW

Reply:
Done